be/src/exprs/agg/distinct.h

```diff
@@ -176,13 +176,11 @@ struct DistinctAggregateStateV2<PT, FixedLengthPTGuard<PT>> {
 
         size_t old_size = set.size();
         src += sizeof(size);
-        const T* data = reinterpret_cast<const T*>(src);
```

Comment:
`reinterpret_cast` just copy bytes to `T* data`, I think the reason is address of src isn't aligned by 32 bytes.